posthog/cache_utils.py

```diff
@@ -11,14 +11,16 @@ def cache_for(cache_time: timedelta):
     def wrapper(fn):
         @wraps(fn)
         @no_type_check
-        def memoized_fn(*args, use_cache=not TEST):
+        def memoized_fn(*args, use_cache=not TEST, **kwargs):
```

Comment:
I learned about python in understanding how this `use_cache` keyword arg allows consumers to care about the caching.

It's a little surprising to me but toggling settings is just as distant from usage so 🤷